test/mocks/MockTransactionClient.ts

```diff
@@ -69,4 +78,19 @@ export class MockedTransactionClient extends TransactionClient {
 
     return txnResponse;
   }
+
+  checkIndividualTransactions(txn: AugmentedTransaction): Promise<TransactionSimulationResult> {
+    // We know it's tryMulticall so there is only one argument.
+    const returnData = txn.args[0].map(([result]) => {
+      return {
+        success: result === txnClientPassResult,
+        data: result,
+      };
+    });
+    return {
+      transaction: { ...txn },
```

Comment:
Does this need to be a new object?

```suggestion
      transaction: txn,
```

Reply:
True. https://github.com/across-protocol/relayer/pull/1761/commits/2a41182e4aac8ecda63b3d399f088efb5826a7c2